Add wear exclusion options to ActiNet

aidanacquah opened the pull request:
Addressing #6, we look to make it possible to run actinet while having the option to exclude first and last days, or days with low wear.

This is unlocked using:
-e ['first', 'last', 'both', None (default)] -> exclude first/last day of wear
-w [time-like string, e.g '30min', '21H'] -> exclude days with wear below threshold

The output wear device metrics, and averages for model outputs are updated to the appropriate wear time.
This PR follows approaches in https://github.com/OxWearables/stepcount

chanshing commented:
Looks good to me!

I see this PR also adds two new fields:
```
    info['ActiNetVersion'] = __version__
    info['ActiNetArgs'] = vars(args)
```

I would mention this change in the PR description, or perhaps split this PR into two.

aidanacquah commented:
> Looks good to me!
> 
> I see this PR also adds two new fields:
> 
> ```
>     info['ActiNetVersion'] = __version__
>     info['ActiNetArgs'] = vars(args)
> ```
> 
> I would mention this change in the PR description, or perhaps split this PR into two.

Removed and added to new PR.
